apps/site/assets/ts/stop/components/DepartureTimes.tsx

```diff
@@ -93,13 +93,14 @@ const departureTimeRow = (
           alertBadgeWrapper(alertClass, alertBadge!)}
         {(hasShuttleService(alerts) ||
           hasSuspension(alerts) ||
-          (hasDetour(alerts) && timeList.length === 0)) && (
-          <>
-            <div style={{ float: "right" }}>See alternatives</div>
-            <br />
-            {alertBadgeWrapper(alertClass, alertBadge!)}
-          </>
-        )}
+          (hasDetour(alerts) && timeList.length === 0)) &&
+          alertBadge && (
+            <>
+              <div style={{ float: "right" }}>See alternatives</div>
+              <br />
+              {alertBadgeWrapper(alertClass, alertBadge!)}
```

Comment:
nit-pick: non-blocking `alertBadge!` doesn't need the `!` since we checked its validity above.